src/main.c

```diff
@@ -256,9 +256,8 @@ static int insertkeys(Sdb *s, const char **args, int nargs, int mode) {
 }
 
 static int createdb(const char *f, const char **args, int nargs) {
-	char *line, *eq;
 	s = sdb_new (NULL, f, 0);
-	if (!s || !sdb_disk_create (s)) {
+	if (!s) {
```

Comment:
uhm, createdb is no longer creating the database? dont we have tests for this?

Reply:
maybe the behaviour is the same, and i didnt chkd but i dont think sdb_new is creating the file if it doesnt exist

Reply:
It will be created in `sdb_sync()` later, so the behavior is the same.

Reply:
Not really because the early creation may be used for locking purposes. But as long as rhere are no tests for this its hard to say the current behaviour is good